pkgs/by-name/py/pyroveil/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  python3,
+  makeWrapper,
+}: stdenv.mkDerivation (final: {
+  pname = "pyroveil";
+  version = "e7d5a7fe3bfd6313509ad6d4cc5d7685c7b2098a";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

Reply:
In this case, as there are no tags, `version` should be `0-unstable-2025-11-17` (the date of the last commit) and, `src` should have `rev = "e1f547372cf1b9d14da56621716d2137088d0061"`.